source/change-streams/tests/change-streams-resume-errorLabels.yml

```diff
@@ -0,0 +1,1134 @@
+# Tests for resume behavior on server versions that support the ResumableChangeStreamError label
+collection_name: &collection_name "test"
+database_name: &database_name "change-stream-tests"
+collection2_name: &collection2_name "test2"
```

Comment:
collection2_name and database2_name don't seem to be referenced in the rest of this file.

Reply:
Removed.